### ISSUE TITLE
README graphic illustration displays some typos

### ISSUE THREAD

**Issue by luzpaz** [OpenCollaborationPlatform/CollaborativeFC#13]
![image](https://user-images.githubusercontent.com/4140247/148378640-786e781a-2145-4c67-94cb-5b9a200ac265.png)

Should be `Rights` but this seems corrected in the source so just the image needs updating.

**Comment by ickby** [comment]
good catch. I update the pictures when merging the "errorhandling" branch, as this one changes some UI elements and hence need redoing the pictures.
